bundles/core/org.eclipse.smarthome.core.thing.xml/src/main/java/org/eclipse/smarthome/core/thing/xml/internal/ThingTypeConverter.java

```diff
@@ -36,6 +37,8 @@
 
     public ThingTypeConverter() {
         super(ThingTypeXmlResult.class, "thing-type");
```

Comment:
Wouldn't it be simpler to call:
<code>this(ThingTypeXmlResult.class, "thing-type");</code>
So, we do not need to add the two new lines on both constructors.